lib/metadataTypes/Asset.js

```diff
@@ -169,7 +169,7 @@ class Asset extends MetadataType {
         const subtypeIds = subTypeArray?.map(
             (subTypeItemName) => Asset.definition.typeMapping[subTypeItemName]
         );
-        const uri = 'asset/v1/content/assets/query';
+        const uri = '/asset/v1/content/assets/query';
```

Comment:
the url from sfmc returns a / at the end of the path so shouldnt be needed?

Reply:
while thats true we seem to be passing in URLs with leading slash everywhere. it breaks our testing suite if a URL passed into your SDK does not start with a slash ;-)

Reply:
ok, so lets change this then